docs/azure/sdk/aspnetcore-guidance.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Best practices for using the Azure SDK with ASP.NET Core
+description: Learn best practices and the steps to properly implement the Azure SDK for .NET in your ASP.NET Core apps.
+ms.topic: conceptual
+ms.custom: devx-track-dotnet
+ms.date: 10/22/2024
+---
+
+# Use the Azure SDK for .NET in ASP.NET Core apps
+
+The Azure SDK for .NET enables ASP.NET Core apps to integrate with many different Azure services. In this article, you'll learn best practices and the steps to implement the Azure SDK for .NET in your ASP.NET Core apps. You'll learn how to:
+
+- Register services for dependency injection.
```

Comment:
nit: I defer to you on this @alexwolfmsft -- I expect a reader who has a good understanding of DI to know that you mean services in the DI "service collection" sense.  But if a reader is not familiar with this terminology and only knows about the Azure client/service terminology, they might think this means they will learn to register an Azure service.  Perhaps something like "register Azure SDK clients with the DI service collection" might resolve this possible ambiguity?